code/game/objects/items/weapons/storage/firstaid.dm

```diff
@@ -102,8 +102,8 @@
 /obj/item/weapon/storage/firstaid/brute/New()
 	..()
 	if (empty) return
-	new /obj/item/weapon/reagent_containers/pill/patch/styptic(src)
-	new /obj/item/weapon/reagent_containers/pill/patch/styptic(src)
+	new /obj/item/stack/medical/gauze(src)

```

Comment:
remove the brute pack thats in brute medkits and replace it with gauze. i put it there exactly for this